### DISCUSSION TITLE
Error при переключении на Russian Language

### DISCUSSION THREAD

**Discussion by azarro** [madromas/esoTalk-v3#2] [Q&A] [answered]
@madromas При переключении на Русский язык, выдает ошибки при переходе в топик.
При обратном переключении на English, ошибок нет.

----------------------------------------------------------------------------------------
![2023-08-08_221201](https://github.com/madromas/esoTalk-v3/assets/2754781/0659d9bf-bdb2-449b-a899-d95a3a0d4207)

**Comment by madromas** [comment]
Сейчас поправим.

**Comment by madromas** [comment, marked as answer]
> 

Пожалуйста удалите папку Russian, и залейте новую с архива.

**Reply by azarro** [reply]
Проверил, теперь ошибки нет.
